### PR TITLE
Add tracing endpoints to opa-ams config & regen templates

### PR DESCRIPTION
**Pull request by ianbillett**
This PR follows the example set [here](https://github.com/observatorium/api/pull/127/files) and adds tracing context to the opa-ams sidecar. 

Signed-off-by: Ian Billett <ibillett@redhat.com>

### CODE REVIEW

File: services/sidecars/opa-ams.libsonnet [hunk 1]
```diff
@@ -77,6 +77,34 @@ function(params) {
...
+                  [
+                    '--internal.tracing.sampling-fraction=' + opa.config.internal.tracing.samplingFraction,
+                  ]
+                else []
+              ) + (
+                if std.objectHas(opa.config.internal.tracing, 'serviceName') then
+                  [
+                    '--internal.tracing.service-name=' + opa.config.internal.tracing.serviceName,
+                  ]
+                else []
```
Comment (periklis) [inline services/sidecars/opa-ams.libsonnet:105]: Do we have defaults for these config options?

**Review by onprem — APPROVED** [review]
LGTM!

File: services/sidecars/opa-ams.libsonnet (hunk 1 above)
Comment (onprem) [reply services/sidecars/opa-ams.libsonnet:105]: Yes, these three opa-ams flags have default values. ref: https://github.com/observatorium/opa-ams/blob/master/main.go#L113-L120

Comment (ianbillett) [reply services/sidecars/opa-ams.libsonnet:105]: The default options for each of these flags is implemented in the `opa-ams` binary itself [here](https://github.com/observatorium/opa-ams/blob/master/main.go#L113-L120). The semantics here are "if the value has been specified in opa.config then add it to the jsonnet with the right flag, else leave it unspecified and use the default from the binary".

**Pull request merged**